### DISCUSSION TITLE
Support "python.autoComplete.showAdvancedMembers" to hide dunder attributes

### DISCUSSION THREAD

**Discussion by StepVK** [microsoft/pylance-release#2638] [Enhancement]
## Environment data

-   Language Server version: 2020.12.2
-   OS and version: Windows 10
-   Python version (& distribution if applicable, e.g. Anaconda): 3.9.1 64bit

## Expected behavior

When I set
JSON: ("python.autoComplete.showAdvancedMembers": false)
GUI: "Python › Auto Complete: Show Advanced Members
        Controls appearance of methods with double underscores in the completion list."
to false, I expect methods and attributes with names starting with "__" to not show up in autocomplete, when called from other scopes.

## Actual behavior
"__" methods and attributes show up in autocomplete regardless of that option.

## Code Snippet / Additional information
![image](https://user-images.githubusercontent.com/41023665/103151269-8c048780-478d-11eb-9050-0a6530c618b4.png)
Seeing _stopped is ok,
__thread shouldn't be there, as shouldn't all the "magic" methods starting with __

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Reply by PolarBean** [reply]
this would be very useful
